### DISCUSSION TITLE
Tooltip width

### DISCUSSION THREAD

**Discussion by antoniomelendez** [saadeghi/daisyui#2562] [Q&A] [answered]
How can i set the width of the tooltip window? I would like it to be wider than the default to avoid really tall tooltips.

**Comment by saadeghi** [comment, marked as answer]
I wouldn't suggest that. Generally you shouldn't even put a long text in as a tooltip in the first place.  
Tooltip in all UIs (not just daisyUI) must be used for small hints. One or a few words. More than that it's not easy to read and probably it's better to write it in the page instead. It also creates responsive issues if the tooltip is wide and contains a lot of text because it won't fit in mobile (and CSS can not dynamically move the tooltip to available space)

But if you really want to it, here's an example:  
https://play.tailwindcss.com/MkDhLLOpyw
